packages/broker/test/integration/plugins/operator/InspectRandomNodeHelper.test.ts

```diff
@@ -0,0 +1,167 @@
+import { JsonRpcProvider, Provider } from "@ethersproject/providers"
+import { Chains } from "@streamr/config"
+import { Wallet } from "@ethersproject/wallet"
+import { parseEther } from "@ethersproject/units"
+import { Logger, TheGraphClient, toEthereumAddress, wait, waitForCondition } from '@streamr/utils'
+import fetch from 'node-fetch'
+
+import type { TestToken } from "@streamr/network-contracts"
+import type { StreamRegistry } from "@streamr/network-contracts"
+
+import { tokenABI } from "@streamr/network-contracts"
+import { streamRegistryABI } from "@streamr/network-contracts"
+import { Contract } from "@ethersproject/contracts"
+
+import { deploySponsorship } from "./deploySponsorshipContract"
+import { createWalletAndDeployOperator } from "./createWalletAndDeployOperator"
+import { InspectRandomNodeHelper } from "../../../../src/plugins/operator/InspectRandomNodeHelper"
+
+const config = Chains.load()["dev1"]
+const theGraphUrl = `http://${process.env.STREAMR_DOCKER_DEV_HOST ?? '10.200.10.1'}:8000/subgraphs/name/streamr-dev/network-subgraphs`
+
+const logger = new Logger(module)
+
+jest.setTimeout(600 * 1000)
+
+describe("InspectRandomNodeHelper", () => {
+    const chainURL = config.rpcEndpoints[0].url
+
+    let provider: Provider
+    let token: TestToken
+    let adminWallet: Wallet
+    let streamId1: string
+    let streamId2: string
+    let graphClient: TheGraphClient
+
+    beforeAll(async () => {
+        provider = new JsonRpcProvider(chainURL)
+        logger.debug("Connected to: ", await provider.getNetwork())
+
+        const streamCreatorKey = "0xfe1d528b7e204a5bdfb7668a1ed3adfee45b4b96960a175c9ef0ad16dd58d728"
+        adminWallet = new Wallet(streamCreatorKey, provider)
+
+        token = new Contract(config.contracts.LINK, tokenABI) as unknown as TestToken
+        const timeString = (new Date()).getTime().toString()
+        const streamPath1 = "/operatorclienttest-1-" + timeString
+        const streamPath2 = "/operatorclienttest-2-" + timeString
+        streamId1 = adminWallet.address.toLowerCase() + streamPath1
+        streamId2 = adminWallet.address.toLowerCase() + streamPath2
+        const streamRegistry = new Contract(config.contracts.StreamRegistry, streamRegistryABI, adminWallet) as unknown as StreamRegistry
+        logger.debug(`creating stream with streamId1 ${streamId1}`)
+        await (await streamRegistry.createStream(streamPath1, "metadata")).wait()
+        logger.debug(`creating stream with streamId2 ${streamId2}`)
+        await (await streamRegistry.createStream(streamPath2, "metadata")).wait()
```

Comment:
Could use StreamrClient to create the streams?
```
const client = createClient(await fetchPrivateKeyWithGas())
const stream1 = await createTestStream(client, module)
const stream2 = await createTestStream(client, module)
await client.destroy()
```

Reply:
This works and after fiddling with the many config types I couldn't get the client to connect to the right chain, so I am leaving it as is.